boards/arm/nucleo_l476rg/nucleo_l476rg.dts

```diff
@@ -63,6 +63,26 @@
 	};
 };
 
+&clk_hsi{
+	status = "okay";
+};
+
+&pll {
+	div-m = <1>;
+	mul-n = <20>;
+	div-r = <4>;
+	clocks = <&clk_hsi>;
+	status = "okay";
+};
+
+&rcc {
+	clocks = <&pll>;
+	clock-frequency = <DT_FREQ_M(80)>;
```

Comment:
I assume this value could be automatically computed given HSI clock value is known as well as PLL formulae. Could be done in the future, though. I'm not sure, though, if it's in general better to specify `clock-frequency` and let the driver figure out the PLL values. Again, likely for future revisions.

Reply:
Yes, I don't see obstacles to get it computed automatically, but for this step I prefer to keep same level of features.